mobile/src/components/home/DiscoverBar.jsx

```diff
@@ -1,17 +1,31 @@
 import React from 'react';
 import {View, Text, StyleSheet, Image, TouchableOpacity, Linking} from 'react-native';
 import {colors, globalStyleSheet} from '../../utilites/Theme';
-
-
+import Svg, { Path } from "react-native-svg"
 const DiscoverBar = () => {
   return (
     <View style={styles.DiscoverRow}>
 
       <TouchableOpacity style={styles.container} onPress={() => Linking.openURL('https://isa.ualberta.ca/')}>
-      <Image
-            source={require('../../../assets/Website.png')}
-            style={styles.image}
-          />
+      
+        <Svg marginTop={6}
```

Comment:
Not sure if it's possible but could we move these svg definitions to their own files? Would make the code cleaner.

Reply:
Should be fixed now

Reply:
LGTM, probably the best we can do unless we get approval to do solid color background.